src/material/select/select.ts

```diff
@@ -598,6 +599,10 @@ export class MatSelect extends _MatSelectMixinBase implements AfterContentInit,
     }
   }
 
+  ngAfterViewChecked(): void {
+    this._changeDetectorRef.markForCheck();
```

Comment:
I'm fairly certain that this will trigger an infinite change detection loop.

Reply:
Not infinite. But often.

Reply:
I'm sorry, but in general we don't sacrifice performance to handle edge cases. We should definitely fix the error, but find a better way to do it.

Reply:
I'm looking for a better way, but it is not so easy. If you use markForCheck on "ngDoCheck" hook it also works. I don't think these changes will damage performance.